packages/appstore/src/modules/wallets/wallets.tsx

```diff
@@ -13,12 +13,17 @@ const Wallets = observer(() => {
     const { client, ui } = useStore();
     const { switchAccount, is_authorize } = client;
     const { is_mobile } = ui;
-    const { data } = useWalletsList();
-    const active_wallet = useActiveWallet();
+    const { data } = useWalletAccountsList();
```

Comment:
@arshad-rao-deriv 
small suggestion
maybe we can rename `data` during destructuring for better readability in the rest of the code? 🤔
smth like:
`const { data: wallets_list } = useWalletAccountsList();`

Reply:
Updated.